src/w3iProxy/index.ts

```diff
@@ -232,6 +233,7 @@ class Web3InboxProxy {
       this.pushClient = await NotifyClient.init({
         identityKeys: this.identityKeys,
         logger: this.logger,
+        // @ts-expect-error - not right now.
```

Comment:
@devceline can you help fix the type errors here please? I think there's also one on `identityKeys` above